src/argilla/utils/modeling.py

```diff
@@ -0,0 +1,241 @@
+#  coding=utf-8
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import re
+from typing import Dict, List, Optional, Tuple, Union
+
+import pandas as pd
+import textdescriptives as td
+from tqdm import tqdm
+
+from argilla.client.feedback.dataset.local.dataset import FeedbackDataset
+from argilla.client.feedback.dataset.remote.dataset import RemoteFeedbackDataset
+from argilla.client.feedback.schemas.metadata import (
+    FloatMetadataProperty,
+    IntegerMetadataProperty,
+    TermsMetadataProperty,
+)
+from argilla.client.feedback.schemas.records import FeedbackRecord
+from argilla.client.feedback.schemas.remote.records import RemoteFeedbackRecord
+
+
+class TextDescriptivesExtractor:
+    """This class extracts a number of basic text descriptives from FeedbackDataset
+    records using the TextDescriptives library and adds them as record metadata."""
+
+    def __init__(
+        self,
+        model: str = "en",
+        metrics: Optional[List[str]] = None,
+        fields: Optional[List[str]] = None,
+        visible_for_annotators: bool = True,
+        progress: bool = True,
```

Comment:
I believe we use `show_progress` as argument for other methods.

Reply:
Changed `progress` to `show_progress`